src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+| MotionChannelCount         | RECOMMENDED           | number        | Number of motion channels included in the recording. A channel corresponds to a time series that has one value per time point. (e.g., a time series of coordinates for positions in 3D consists of 3 channels, each corresponding to the x, y, and z coordinates, respectively.)                                                                                                                                                                                                                                                 |
+| RotationRule               | RECOMMENDED           | string        | In case orientation channels are present, indicate whether rotations are applied clockwise around an axis when seen from the positive direction (left-hand rule) or counter-clockwise (right-hand rule). Must be one of: “left-hand”, “right-hand”.                                                                                                                                                                                                                                                                              |
+| RotationOrder              | RECOMMENDED           | string        | Specify the sequence in which the elemental rotations are applied around the extrinsic axes. A 3D rotation is represented about 3 different axes (Tait-Bryan convention). Must be one of: “XYZ”, ”XZY”, ”YXZ”, “YZX”, “ZXY”, “ZYX”.                                                                                                                                                                                                                                                                                              |
+| SamplingFrequencyEffective | REQUIRED              | number        | Effective sampling rate of the tracking system in Hz. If duration of the corresponding recording is available, effective sampling rate is computed by dividing the `RecordingDuration` in seconds by the number of samples included in the time series. If not available, the field takes the same value as field `SamplingFrequency`.                                                                                                                                                                                           |
+| SoftwareVersions           | RECOMMENDED           | string        | Manufacturer’s designation of the acquisition software.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
move up, with the general fields shared between modalities

Reply:
It is a recommended field, therefore we left it in the corresponding table.